### PR TITLE
MUI Integration Instructions

### PR DESCRIPTION
**Pull request by spreston4**
Updated README with instructions for using the `mui` option for Shipwright Tokens and integrating the styles into your project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,4 +103,72 @@ plugins: [
...
+        <Component {...pageProps} />
+      </ThemeProvider>
+```
+### Pick and Choose Styles
+You can pick and choose which styles are availble in your project by editing the generated `muiTheme.js` to only include the styles you want by adding/removing the following snippets:
+
+### Colors:
+- Import your generated `colors.json` file:
+```
+const colors = require("./yourOutputDirectory/colors");
```
Comment (Ganzfriend) [inline README.md:138]: We could also just call this palette, directly.

```suggestion
const palette = require("./yourOutputDirectory/colors");
```

That way, we'd just add it directly into the createTheme object, the same way we do with shadows.

File: README.md [hunk 2]
```diff
@@ -102,4 +103,72 @@ plugins: [
...
+```
+- Include `palette: colors` as a key/value pair in the object passed to the `createTheme` function:
+```
+const theme = createTheme({
+  palette: colors,
```
Comment (Ganzfriend) [inline README.md:143]:
```suggestion
  palette,
```

**Review by Ganzfriend — COMMENTED** [review]
This looks awesome! The one nit is to correct something I didn't do very cleanly the first time through.

**Comment by spreston4** [comment]
@Ganzfriend Good call! I updated the `muiTheme.js` files in the example project, and in our `theme_templates` folder to match. Though that would be good for the sake of consistency.

**Review by Ganzfriend — APPROVED** [review]
Awesome work, Sam!

**Pull request merged**